static/js/lib/queries/enrollment.js

```diff
@@ -32,3 +32,33 @@ export const deactivateEnrollmentMutation = (enrollmentId: number) => ({
     }
   }
 })
+
+export const courseEmailsSubscriptionMutation = (
+  enrollmentId: number,
+  emailsSubscription: string = ""
+) => ({
+  url:  `/api/enrollments/${enrollmentId}/`,
+  body: {
+    receive_emails: emailsSubscription
+  },
+  options: {
+    ...getCsrfOptions(),
+    method: "PATCH"
+  },
+  update: {
+    enrollments: prevEnrollments => {
+      if (!emptyOrNil(prevEnrollments)) {
+        if (emailsSubscription) {
+          prevEnrollments.find(
+            enrollment => enrollment.id === enrollmentId
+          ).emails_subscription = true
+        } else {
+          prevEnrollments.find(
+            enrollment => enrollment.id === enrollmentId
+          ).emails_subscription = false
+        }
```

Comment:
We can probably simplify this expression into lesser lines:
```suggestion
          prevEnrollments.find(
            enrollment => enrollment.id === enrollmentId
          ).emails_subscription = emailsSubscription
```